distribution/docs/src/main/jdocs/content/_security-framework/security-ldap-embedded-server-contents.adoc

```diff
@@ -0,0 +1,367 @@
+:title: Embedded LDAP Server
+:type: subSecurityFramework
+:status: published
+:parent: Security LDAP
+:order: 00
+:summary: Embedded LDAP server.
+
+${branding} includes an embedded LDAP server (OpenDJ) for testing and demonstration purposes.
+
+===== Installing the Embedded LDAP Server
+
+The embedded LDAP server is not installed by default with a standard installation.
+
+. Navigate to the ${admin-console}.
+. Select *Manage* Applications.
+. Install the *OpenDJ Embedded* application.
+
+===== Configuring the Embedded LDAP
+
+Configure the Embedded LDAP from the ${admin-console}:
+
+. Navigate to the *${admin-console}*.
+. Select *Manage* Applications.
+. Start the *OpenDj Embedded* application.
+. Once started, select the *OpenDj* application.
+. Select the *Configuration* tab.
+
+.OpenDJ Embedded Configurable Properties
+[cols="1,7" options="header"]
+|===
+
+|Configuration Name
+|Description
+
+|LDAP Port
+|Sets the port for LDAP (plaintext and startTLS). 0 will disable the port.
+
+|LDAPS Port
+|Sets the port for LDAPS. 0 will disable the port.
+
+|Base LDIF File
+|Location on the server for a LDIF file.
+This file will be loaded into the LDAP and overwrite any existing entries.
+This option should be used when updating the default groups/users with a new LDIF file for testing.
+The LDIF file being loaded may contain any LDAP entries (schemas, users, groups, etc.).
+If the location is left blank, the default base LDIF file will be used that comes with ${branding}.
+
+|===
+
+===== Trust Certificates
+
+For LDAPS or startTLS to function correctly, it is important that the LDAP server is configured with a keystore file that trusts the clients it is connecting to and vice versa.
+Complete the following procedure to provide your own keystore information for the LDAP. 
```

Comment:
There is no procedure to provide keystore information for the LDAP since it now uses the default key/truststore for DDF. We can remove this section.

Reply:
removed